src/App.vue

```diff
@@ -238,9 +285,49 @@ export default Vue.extend({
         // Ignore passed value argument.
         this.$store.commit("setUnauthorizedError", null);
       }
+    },
+    availableProjects() {
+      return this.$store.state.projects.projects;
+    },
+    ...mapGetters({
+      project: "projects/getProjectById"
+    }),
+    sensibleColors() {
+      let obj = {};
+      for (let color in colors) {
+        if (["yellow", "shades"].indexOf(color) === -1) {
+          for (let shade in colors[color]) {
+            if (shade.includes("base") || shade.includes("darken2")) {
+              obj[String(color + "_" + shade)] = colors[color][shade];
+            }
+          }
+        }
+      }
+      return obj;
+    }
+  },
+  methods: {
+    setActiveProject(projectID) {
+      this.$store.commit("setCurrentlyActiveProject", projectID);
+      this.$vuetify.theme.primary = this.projectPrimaryColor(projectID);
+    },
+    returnToDefault() {
+      this.$vuetify.theme.primary = colors.blue.base;
+      this.$store.commit("setCurrentlyActiveProject", null);
+    },
+    projectPrimaryColor(projectID) {
+      const sortedColors = Object.values(this.sensibleColors).sort();
+      return String(
+        sortedColors[projectID % Object.keys(this.sensibleColors).length]
+      );
+    },
+    camelToKebab(string) {
```

Comment:
According to WebStorm, this method is not used anywhere. Are you planning to add functionality?

Reply:
Oops no, this is a left-over from a previous commit. I'll remove it.